ipatests/test_integration/test_commands.py

```diff
@@ -679,3 +679,51 @@ def test_user_mod_change_capitalization_issue5879(self):
         assert 'Modified user "%s"' % (ipauser) in cmd.stdout_text
         assert 'First name: %s' % (modfirst) in cmd.stdout_text
         assert 'Last name: %s' % (modlast) in cmd.stdout_text
+
+    def test_ipa_healthcheck_log_file_naming_issue35(self):
+        """
+        Test that when ipa-healthcheck.timer command is started
+        on IPA server the log file created under in
+        /var/log/ipa/healthcheck contains healthcheck.log file
+        This is a testcase for Pagure issue
+        https://github.com/freeipa/freeipa-healthcheck/issues/35
+
+        Steps:
+        1. Setup IPA server and install ipa-healthcheck tool
+        2. Enable ipa-healthcheck.timer service
+        3. Start ipa-healthcheck.timer service
+        4. Check /var/log/ipa/healthcheck directory for log file
+        """
+        log_file = '/var/log/ipa/healthcheck/healthcheck.log'
+        systemd_healthcheck_file = '/etc/systemd/system/' \
+                                   'multi-user.target.wants/' \
+                                   'ipa-healthcheck.timer'
+        # Install ipa-healthcheck tool
+        tasks.kinit_admin(self.master)
+        result = self.master.run_command(['dnf', 'install', '-y',
```

Comment:
dnf command will not be available on RHEL 7.x. You can use `which dnf >/dev/null` and check the return code.
I would advise to add a new helper function in ipatests/pytest_ipa/integration/tasks.py, for instance
`def install_package(host, pkgs)`.